gno.land/pkg/gnoweb/alias.go

```diff
@@ -25,3 +27,21 @@ var Redirects = map[string]string{
 	"/getting-started":         "/start",
 	"/gophercon24":             "https://docs.gno.land",
```

Comment:
We can remove this I think

Reply:
At some point, we should add a param to specify whether the redirects are permanent or temporary.

Reply:
removed